pep-0695.rst

```diff
@@ -820,13 +834,10 @@ Library Changes
 
 Several classes in the ``typing`` module that are currently implemented in
 Python must be reimplemented in C. This includes: ``TypeVar``,
-``TypeVarTuple``, ``ParamSpec``, ``Generic``, and ``Union``. The new class
+``TypeVarTuple``, ``ParamSpec``, and ``Generic``. The new class
 ``TypeAliasType`` (described above) also must be implemented in C. The
 documented behaviors of these classes should not change.
```

Comment:
Didn't we agree to leave some of this in Python? This paragraph uses "must" which feels that would violate the PEP (even though we heard from Emily that this isn't an SC requirement).

Reply:
Good point, I'll make this less definite.